### ISSUE TITLE
setup-travis-yml.pl --create doesn't work without a travis.yml

### ISSUE THREAD

**Issue by 2shortplanks** [maxmind/App-CISetup#5]
In a new project:

```
prompt$ setup-travis-yml.pl --dir . --create
Error open (<:unix:utf8_strict) on '.travis.yml': No such file or directory at /opt/markperl/lib/site_perl/5.22.0/App/CISetup/Role/ConfigUpdater.pm line 114.
```

**Comment by autarch** [comment]
I swear I'd done that at some point, but hmm. This thing needs more tests, clearly!

**Comment by autarch** [comment]
Fixed in v0.07. I tested by hand like the scrub I am.
